src/index.jsx

```diff
@@ -103,27 +103,38 @@ function bootstrap() {
 function render(Component) {
   const root = document.getElementById('react');
   const { configuration } = store.getState();
-  const googleAuthority = 'https://accounts.google.com';
-  // TODO once we no longer query BQ directly, we'll no longer need the BQ scope
+  const isGoogleAuthority = configuration.configObject.authorityEndpoint.startsWith('https://accounts.google.com');
```

Comment:
Which format does the new authority endpoint take that we can no longer use strict equality?

Reply:
Oh it could have a slash at the end